spec/std/process/status_spec.cr

```diff
@@ -226,6 +226,16 @@ describe Process::Status do
       assert_prints Process::Status.new(exit_status(255)).to_s, "255"
     end
 
+    it "on abnormal exit" do
+      {% if flag?(:win32) %}
+        assert_prints status_for(:interrupted).to_s, "3221225786"
+      {% elsif flag?(:wasi) %}
+        assert_prints status_for(:interrupted).to_s, "INT"
+      {% else %}
+        assert_prints status_for(:interrupted).to_s, "INT"
+      {% end %}
```

Comment:
Why not just leave the `else` branch?

ditto below

Reply:
Argh, thanks for noticing. I meant to delete the wasi branches because the spec doesn't execute there anyway.